test/test_cpp_extensions_aot.py

```diff
@@ -270,6 +270,23 @@ def _run_identity(prior_mem, device):
             curr_mem = torch.cuda.memory_allocated(device)
             self.assertEqual(curr_mem, init_mem)
 
+        # (3) test calling our dispatcher on ones_like
+        t = torch.rand(32, 16, device=device)
+        cpu_t = libtorch_agnostic.ops.my_ones_like(t, "cpu")
+        self.assertEqual(cpu_t, torch.ones_like(t, device="cpu"))
+
+        def _make_cuda_tensors(prior_mem):
+            cuda_t = libtorch_agnostic.ops.my_ones_like(t, "cuda")
```

Comment:
note, passing in the device obj here does not work
```suggestion
            cuda_t = libtorch_agnostic.ops.my_ones_like(t, device)
```